app/src/main/java/org/sopt/and/ui/signup/SignUpScreen.kt

```diff
@@ -64,95 +61,56 @@ fun SignUpScreen(
         topBar = { SignUpTopBar() },
         bottomBar = {
             SignUpBottomBar(
-                userInfo = userInfo,
                 isSignUpButtonEnabled = viewModel.isSignUpButtonEnabled(),
-                onSignUpClick = {
-                    viewModel.validateAndHandleEmailPassword { validUserInfo ->
-                        navigationToSignIn(validUserInfo)
-                    }
-                }
+                onSignUpClick = { viewModel.signUp() }
```

Comment:
```suggestion
                onSignUpClick = viewModel::signUp
```
이런것도 있는거 알고 계신가요?? 제가 완전 좋아하는 방법입니다🤭